### PR TITLE
fix #1399

### LINKED ISSUE #1399: Overlapping of footer message with the tab details on the middle pane
debugger_issue1

'First time connecting to Firefox?' message is overlapping with the tabs shown on the page.

### PR DESCRIPTION
**Pull request by leibale**
Associated Issue: #1339 

### Summary of Changes

* Changed footer to **not** be position absolute

### Test Plan

- [x] Open `localhost:8000`
- [x] Check if that [this](https://cloud.githubusercontent.com/assets/3799600/20978395/c38a7af0-bcce-11e6-8e09-a35496e7056e.JPG) not happens

@jasonLaster or @jlongster please review :)

### CODE REVIEW

**Comment by jasonLaster** [comment]
Thanks @leibale!

Could we keep the footer on the bottom, without overlaying the tabs?

**Comment by leibale** [comment]
By "bottom" you mean bottom of the window (and not bottom of the page)?

**Comment by jasonLaster** [comment]
yup - bottom of the window

**Comment by leibale** [comment]
Will do it soon

File: packages/devtools-local-toolbox/src/components/LandingPage.css [hunk 1]
```diff
@@ -56,7 +56,7 @@
   display: flex;
   flex: 1;
   height: 100%;
-  overflow: auto;
+  /*overflow: auto;*/
```
Comment (jasonLaster) [inline packages/devtools-local-toolbox/src/components/LandingPage.css]: lets delete this

File: packages/devtools-local-toolbox/src/components/LandingPage.css [hunk 2]
```diff
@@ -92,13 +93,17 @@
 }
 
 .landing-page .tab-group {
+  position: absolute;
   margin: 40px;
+  height: calc(100% - 250px);
+  width: calc(100% - 80px);
+  overflow: auto;
```
Comment (jasonLaster) [inline packages/devtools-local-toolbox/src/components/LandingPage.css]: will this scroll?

File: packages/devtools-local-toolbox/src/components/LandingPage.css [hunk 3]
```diff
@@ -92,13 +93,17 @@
...
 }
 
 .landing-page .tab-list {
   list-style: none;
-  padding: 0px;
-  margin: 0px;
+  padding: 0;
+  margin: 0;
```
Comment (jasonLaster) [inline packages/devtools-local-toolbox/src/components/LandingPage.css]: hmm, i think we want the units

File: packages/devtools-local-toolbox/src/components/LandingPage.css (hunk 2 above)
Comment (leibale) [inline packages/devtools-local-toolbox/src/components/LandingPage.css]: if there is a lot of tabs - yes

**Pull request merged**